main.tf

```diff
@@ -17,6 +17,7 @@
 locals {
   seed_project_id             = var.project_id != "" ? var.project_id : format("%s-%s", var.project_prefix, "seed")
   state_bucket_name           = var.state_bucket_name != "" ? var.state_bucket_name : format("%s-%s-%s", var.project_prefix, "tfstate", random_id.suffix.hex)
+  storage_bucket_location     = var.storage_bucket_location != null ? var.storage_bucket_location : var.default_region
```

Comment:
It looks like we use `default_region` only for the bucket within this module? What is the reason for adding an additional `storage_bucket_location`?

Reply:
For high availability of the bucket.  Many customers do not want any of their terraform state subject to a regional disaster such as say an earthquake or any other kind of disaster.

Reply:
@michaeldeman got it, but we can still pass in a multi region location via `default_region` although the name is confusing. Perhaps we should deprecate `default_region` in the next major release while supporting `default_region` and `storage_bucket_location` in the interim. 
I think it also makes sense for the defaults to be multi regional + multi region location.
/cc @rjerrems